### PR TITLE
Add cypress com_banners test

### PR DESCRIPTION
**Pull request by charvimehradu**
Pull Request for adding cypress test for banners.

@rajputanuj31 @laoneo helped a lot thankyou!

### CODE REVIEW

File: tests/System/integration/site/components/com_banner/banner.cy.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+describe('Test that the front page', () => {
+    it('can display banners', () => {
+      cy.db_createBanner({ custombannercode: 'automated test banner 1',type: 1})
+        .then(() => cy.db_createBanner({ custombannercode: 'automated test banner 2' ,type: 1}))
+        .then(() => cy.db_createBanner({ custombannercode: 'automated test banner 3' ,type: 1}))
+        .then(() => cy.db_createBanner({ custombannercode: 'automated test banner 4' ,type: 1}))
+        .then(() => cy.db_createModule({ title: 'automated test',module: 'mod_banners',position: 'sidebar-right',params:'{"count":5}'}))
```
Comment (laoneo) [inline tests/System/integration/site/components/com_banner/banner.cy.js]: the position attribute can be removed as it is a default value

Comment (charvimehradu) [reply tests/System/integration/site/components/com_banner/banner.cy.js]: Hope it is fine now!

File: tests/System/integration/site/modules/mod_banners/Banners.cy.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+describe('Test that the front page', () => {
```
Comment (laoneo) [inline tests/System/integration/site/modules/mod_banners/Banners.cy.js]: Can you change this to something like "Test that the banners module"

File: tests/System/integration/site/modules/mod_banners/Banners.cy.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  it('can display banners', () => {
+    cy.db_createBanner({ custombannercode: 'automated test banner 1', type: 1 })
+      .then(() => cy.db_createBanner({ custombannercode: 'automated test banner 2', type: 1 }))
```
Comment (laoneo) [inline tests/System/integration/site/modules/mod_banners/Banners.cy.js]: Can you move the type: 1 option to the createBanner function as default value.

**Comment by laoneo** [comment]
There are still some cs issues and can you rename the file to Default.cy.js.

File: tests/System/integration/site/modules/mod_banners/Banners.cy.js (hunk 2 above)
Comment (charvimehradu) [reply tests/System/integration/site/modules/mod_banners/Banners.cy.js]: done

File: tests/System/integration/site/modules/mod_banners/Banners.cy.js (hunk 3 above)
Comment (charvimehradu) [reply tests/System/integration/site/modules/mod_banners/Banners.cy.js]: done

**Comment by laoneo** [comment]
Thanks a lot!

**Comment by charvimehradu** [comment]
Glad I could help

**Pull request merged**